src/main/java/org/craftercms/core/service/Content.java

```diff
@@ -39,6 +39,6 @@
     /**
      * Returns a {@link InputStream} for the data.
      */
-    public InputStream getInputStream() throws IOException;
+    InputStream getInputStream() throws IOException;
```

Comment:
Why remove `public`?

Reply:
You don't need to specify `public` in interfaces, all methods are public